public/fonts/index.css

```diff
@@ -0,0 +1,142 @@
+@font-face {
+  font-family: "Noe Display";
+  src: url("./noe-display/NoeDisplay-Medium.eot"); /* IE9 Compat Modes */
+  src: url("./noe-display/NoeDisplay-Medium.eot?#iefix")
+      format("embedded-opentype"),
+    /* IE6-IE8 */ url("./noe-display/NoeDisplay-Medium.otf") format("opentype"),
+    /* Open Type Font */ url("./noe-display/NoeDisplay-Medium.svg")
+      format("svg"),
+    /* Legacy iOS */ url("./noe-display/NoeDisplay-Medium.ttf")
+      format("truetype"),
+    /* Safari, Android, iOS */ url("./noe-display/NoeDisplay-Medium.woff")
+      format("woff"),
+    /* Modern Browsers */ url("./noe-display/NoeDisplay-Medium.woff2")
+      format("woff2"); /* Modern Browsers */
+  font-weight: 500;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Noe Display";
+  src: url("./noe-display/noe-display-bold.eot"); /* IE9 Compat Modes */
+  src: url("./noe-display/noe-display-bold.eot?#iefix")
+      format("embedded-opentype"),
+    /* IE6-IE8 */ url("./noe-display/noe-display-bold.otf") format("opentype"),
+    /* Open Type Font */ url("./noe-display/noe-display-bold.svg") format("svg"),
+    /* Legacy iOS */ url("./noe-display/noe-display-bold.ttf")
+      format("truetype"),
+    /* Safari, Android, iOS */ url("./noe-display/noe-display-bold.woff")
+      format("woff"),
+    /* Modern Browsers */ url("./noe-display/noe-display-bold.woff2")
+      format("woff2"); /* Modern Browsers */
+  font-weight: 700;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Noe Display";
+  src: url("./noe-display/noe-display-bold-italic.eot"); /* IE9 Compat Modes */
+  src: url("./noe-display/noe-display-bold-italic.eot?#iefix")
+      format("embedded-opentype"),
+    /* IE6-IE8 */ url("./noe-display/noe-display-bold-italic.otf")
+      format("opentype"),
+    /* Open Type Font */ url("./noe-display/noe-display-bold-italic.svg")
+      format("svg"),
+    /* Legacy iOS */ url("./noe-display/noe-display-bold-italic.ttf")
+      format("truetype"),
+    /* Safari, Android, iOS */ url("./noe-display/noe-display-bold-italic.woff")
+      format("woff"),
+    /* Modern Browsers */ url("./noe-display/noe-display-bold-italic.woff2")
+      format("woff2"); /* Modern Browsers */
+  font-weight: 700;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: "Camphor";
+  font-weight: 700;
+  font-style: normal;
+  src: url("./Camphor-Bold.eot"); /* IE9 Compat Modes */
+  src: url("./Camphor-Bold.eot?#iefix") format("embedded-opentype"),
+    /* IE6-IE8 */ url("./Camphor-Bold.woff") format("woff"),
+    /* Modern Browsers */ url("./Camphor-Bold.ttf") format("truetype"),
+    /* Safari, Android, iOS */
+      url("./Camphor-Bold.svg#e37ef20b6d73827754050f31838d5c3f") format("svg"); /* Legacy iOS */
```

Comment:
what is this hash?

Reply:
This font css was from the old `SCSS` file and and I believe that hash is required for `svg font` to work on iPhone or iPad. https://web.archive.org/web/20150102141618/http://www.fluorcow.com/embedding-fonts-on-a-website

Reply:
I looked into it a little too. Seems like it's still necessary to keep that identifier.